src/components/ComputeModal.js

```diff
@@ -188,6 +192,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     const runtimeConfig = desiredRuntime && {
       cloudService: desiredRuntime.cloudService,
       ...(desiredRuntime.cloudService === cloudServices.GCE ? {
+        zone: desiredRuntime.zone.toLowerCase(),
```

Comment:
The issue is the Google Cloud Storage API returns locations in uppercase, but the Leonardo APIs accept locations in lowercase. 

I had designed the Leonardo APIs to accept lowercase locations because the GCE metadata APIs return locations in lowercase 👎 . So we can blame Google for the inconsistency. 

Probably the best thing for us to do is update the Leonardo APIs to be case insensitive with locations. This would be fairly easy to implement. There's never going to be a time when US-CENTRAL1 is different from us-central1. Then the UI code here wouldn't need to convert to lowercase.